src/config.ts

```diff
@@ -66,21 +64,41 @@ export class JuvixConfig {
   readonly judocDir = new VsCodeSetting('juvix-mode.opts.judocDir');
 
   public getInternalBuildDir(): string | undefined {
-    if (juvixBuildDir) return juvixBuildDir;
+
+    const useTmpDir = () => {
+      const tmpPath = path.join(tmpdir(), '.juvix-build');
+      try {
+        const tmp = fs.mkdtempSync(tmpPath);
+        const juvixBuildDir = tmp.toString();
+        console.log("TMP dir", juvixBuildDir);
```

Comment:
You can use debugChannel.info here, so we can get all the logging info in the "output" panel.